django_celery_monitor/admin.py

```diff
@@ -191,6 +194,25 @@ def kill_tasks(self, request, queryset):
                 revoke(state.task_id, connection=connection,
                        terminate=True, signal='KILL')
 
+    @action(_('Retry selected tasks'))
+    def retry_tasks(self, request, queryset):
+        with current_app.default_connection() as connection:
+            for state in queryset:
+                module_name, task_name = state.name.rsplit('.', 1)
```

Comment:
This should use `celery.utils.imports.symbol_by_name` which Celery uses. It's effectively importing kombu's utility of the same name, but it's good to stick with the Celery version of it.